docs/snakefiles/reporting.rst

```diff
@@ -246,36 +254,77 @@ Further, using ``url_args()`` you can add URL arguments and using ``anchor()`` y
             file=f,
         )
 
+.. _snakefiles-rendering_reports:
+
 Rendering reports
 -----------------
 
-To create the report simply run
+All the metadata contained in the report (e.g. runtime statistics) is automatically collected during the rendering of the report.
+These statistics are obtained from the metadata that is stored in the ``.snakemake`` directory inside your working directory.
+
+.. _snakefiles-self_contained_html_file:
+
+Self-contained HTML file
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+To create a default report simply run
 
 .. code-block:: bash
 
-    snakemake --report report.html
+    snakemake --report
 
-after your workflow has finished.
-All other information contained in the report (e.g. runtime statistics) is automatically collected during creation.
-These statistics are obtained from the metadata that is stored in the ``.snakemake`` directory inside your working directory.
+after your workflow has finished successfully.
+This will implicitly assume a report file name of ``report.html`` and will create this self-contained HTML report.
+If you want to give your report a custom name, you can do so by specifying this on the command-line:
+
+.. code-block:: bash
+
+    snakemake --report my_report.html
+
+As specifying an HTML file for the report will always embed any user-defined report outputs into the same HTML file, this report output type is only suitable for smaller reports.
 
+.. _snakefiles-self_contained_zip_archive:
 
-You can define an institute specific stylesheet with:
+Self-contained ZIP archive
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For anything more complex, it is recommended to generate a ``ZIP`` archive report.
+For example, if you have lots of samples you report on, or if you specified lots of different workflow outputs for report inclusion.
+To get such a self-contained ``ZIP`` archive, simply specify a ``.zip`` file name instead:
 
 .. code-block:: bash
 
-    snakemake --report report.html --report-stylesheet custom-stylesheet.css
+    snakemake --report some_report.zip
 
-In particular, this allows you to e.g. set a logo at the top (by using CSS to inject a background for the placeholder ``<div id="brand">``, or overwrite colors.
-For an example custom stylesheet defining the logo, see :download:`here <../../tests/test_report/custom-stylesheet.css>`.
-The report for above example can be found :download:`here <../../tests/test_report/expected-results/report.html>` (with a custom branding for the University of Duisburg-Essen).
-The full example source code can be found `here <https://github.com/snakemake/snakemake/tree/main/tests/test_report/>`__.
+You can move this file wherever you need it and then unpack it.
```

Comment:
You can move this file to wherever you want to view it and then unpack it.

??

Reply:
much better, indeed!